MyPerfectOnboardingApplication/MyPerfectOnboarding.Api.Tests/Utils/Extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace MyPerfectOnboarding.Api.Tests.Utils
+{
+    public static class Extensions
+    {
+        public static async Task<HttpResponseMessage> ExecuteAction(this IHttpActionResult response)
+        {
+            return await response.ExecuteAsync(CancellationToken.None);
```

Comment:
You can take this a little bit further! How about [executing the action](https://docs.microsoft.com/en-us/dotnet/api/system.func-2?view=netframework-4.7.2) from controller here?
Therefore in the tests you would only call:
`
var response = await _listController.ExecuteAction(controller => controller.GetAsync());
`
instead of having two (almost)identical lines of code in every test:
````
var response = await _listController.GetAsync();
var message = await response.ExecuteAction();
````